modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java

```diff
@@ -95,26 +100,24 @@
 
 
     private SlotManagerClusterMode() {
-        if (AndesContext.getInstance().isClusteringEnabled()) {
 
-            HazelcastAgent hazelcastAgent = HazelcastAgent.getInstance();
+        HazelcastAgent hazelcastAgent = HazelcastAgent.getInstance();
 
-            //Initialize distributed maps used in this class
-            unAssignedSlotMap = hazelcastAgent.getUnAssignedSlotMap();
-            overLappedSlotMap = hazelcastAgent.getOverLappedSlotMap();
-            slotIDMap = hazelcastAgent.getSlotIdMap();
-            queueToLastAssignedIDMap = hazelcastAgent.getLastAssignedIDMap();
-            nodeToLastPublishedIDMap = hazelcastAgent.getLastPublishedIDMap();
-            slotAssignmentMap = hazelcastAgent.getSlotAssignmentMap();
+        //Initialize distributed maps used in this class
+        unAssignedSlotMap = hazelcastAgent.getUnAssignedSlotMap();
+        overLappedSlotMap = hazelcastAgent.getOverLappedSlotMap();
+        slotIDMap = hazelcastAgent.getSlotIdMap();
+        queueToLastAssignedIDMap = hazelcastAgent.getLastAssignedIDMap();
+        nodeToLastPublishedIDMap = hazelcastAgent.getLastPublishedIDMap();
+        slotAssignmentMap = hazelcastAgent.getSlotAssignmentMap();
 
-            nodeInformedSlotDeletionSafeZones = new HashMap<String, Long>();
+        nodeInformedSlotDeletionSafeZones = new HashMap<String, Long>();
 
-            //start a thread to calculate slot delete safe zone
-            //TODO: use a common  thread pool for tasks like this?
-            slotDeleteSafeZoneCalc = new SlotDeleteSafeZoneCalc(SAFE_ZONE_EVALUATION_INTERVAL);
-            new Thread(slotDeleteSafeZoneCalc).start();
+        //start a thread to calculate slot delete safe zone
+        //TODO: use a common  thread pool for tasks like this?
+        slotDeleteSafeZoneCalc = new SlotDeleteSafeZoneCalc(SAFE_ZONE_EVALUATION_INTERVAL);
+        new Thread(slotDeleteSafeZoneCalc).start();
```

Comment:
This is done at the shutDownSlotManager method which is invoked as part of server shutdown.